terminal/TerminalServer.cpp

```diff
@@ -59,18 +63,13 @@ void runTerminal(shared_ptr<ServerClientConnection> serverClientState,
     // Data structures needed for select() and
     // non-blocking I/O.
     fd_set rfd;
-    fd_set wfd;
-    fd_set efd;
     timeval tv;
 
     FD_ZERO(&rfd);
-    FD_ZERO(&wfd);
-    FD_ZERO(&efd);
     FD_SET(masterfd, &rfd);
-    FD_SET(STDIN_FILENO, &rfd);
```

Comment:
Ah, you are right.  Thank you so much for your PR.  I also have a fix where the single select checks the fd in UnixSocketHandler but it wasn't giving me less CPU cycles on the server, and now I know why :-)